package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "eslint-config-eko",
-  "version": "0.2.3",
+  "version": "1.0.0",
```

Comment:
Suggest version 0.9

Reply:
@tavurth  I though to use `1.0.0` as there are breaking changes. And update will require manual update configurations